strings.js

```diff
@@ -0,0 +1,51 @@
+// 1. Uppercase the first character
+
+function ucFirst(str) {
+    str = str[0].toUpperCase() + str.slice(1);
+    return str
+}
+
+let text = 'text';
+console.log(ucFirst(text));
+
+// 2. Check for spam
+
+function checkSpam(str, checkStr1, checkStr2) {
```

Comment:
it should have one argument, the string that you should check

Reply:
done